lib/mongoid/paranoia.rb

```diff
@@ -18,8 +19,31 @@ module Paranoia
     include Mongoid::Persistable::Deletable
     extend ActiveSupport::Concern
 
+    class << self
+      attr_accessor :configuration
+    end
+
+    def self.configuration
+      @configuration ||= Configuration.new
+    end
+
+    def self.reset
```

Comment:
Added spec for Mongoid::Paranoia.reset, it's a little odd due to how the configuration gets set when included into the defined class - you'll see in the spec

Also cleaned up the configuration_spec so it's more readable.